site-structure.json

```diff
@@ -77,7 +76,8 @@
                     { "pageName": "Panels", "href": "/Panels.html" },
                     { "pageName": "Composite Panels", "href": "/CompositePanels.html" },
                     { "pageName": "Conditional Subpanels", "href": "/ConditionalSubpanels.html" },
-                    { "pageName": "Localization in the Preferences Framework", "href": "/LocalizationInThePreferencesFramework.html" }
+                    { "pageName": "Localization in the Preferences Framework", "href": "/LocalizationInThePreferencesFramework.html" },
+                    { "pageName": "Using String Templates", "href": "/tutorial-usingStringTemplates/UsingStringTemplates.html" }
```

Comment:
I think this String Templates tutorial should probably stay in the Tutorials section, @jhung – but in the Prefs Framework section of the tutorials instead of the UIO section.